R/constants.R

```diff
@@ -116,6 +116,10 @@ aglu.BIO_GRASS_YIELD_KGCM2 <- 0.34
 aglu.BIO_TREE_FERT_IO_GNM2 <- 3.36
 aglu.BIO_TREE_YIELD_KGCM2 <- 0.345
 
+# Water characteristics for biomass (from Vaibhav's paper)
```

Comment:
Tried finding the paper but not sure what it is exactly. Could @pkyle or @pralitp help?